### ISSUE TITLE
Compressed data ended before the end-of-stream marker was reached

### ISSUE THREAD

**Issue by giuse88** [giuse88/duka#10]
```
2016-05-23 07:10:59,588 - INFO - Thread-4 [4460486656] -  Fetching EURUSD 2015 03 23 23h_ticks.bi5
ERROR for 2015-04-16, EURUSD Exception : Compressed data ended before the end-of-stream marker was reached
2016-05-23 07:10:59,661 - INFO - Thread-2 [4449976320] -  Day 2015-04-16 fetched in 9.455474853515625s
2016-05-23 07:10:59,662 - INFO - Thread-2 [4449976320] -  Fetching day 2015-04-24
2016-05-23 07:10:59,667 - INFO - Thread-2 [4449976320] -  Fetching EURUSD 2015 03 24 00h_ticks.bi5
2016-05-23 07:10:59,669 - INFO - Thread-2 [4449976320] -  Fetching EURUSD 2015 03 24 01h_ticks.bi5
```

**Comment by kenorb** [comment]
Fixed by faf082ec95497ec0dea65bf3b7564101ca6a12db?

**Comment by giuse88** [comment]
@kenorb Yes, it does as far as I can say. The exception disappeared and the data is not corrupted.  I tried we a few binary blocks and it always worked.
